TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift

```diff
@@ -28,9 +30,14 @@ class AudioRecordingsTableViewController: UITableViewController {
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let currentAudio = AudioManager.sharedInstance.getRecordingForIndex(index: indexPath.row)
         
-        let cellText = currentAudio.fileName
-        let audioCell = tableView.dequeueReusableCell(withIdentifier: "audio", for: indexPath)
-        audioCell.textLabel?.text = cellText
+        let cellFileName = currentAudio.fileName
+        let tagName = currentAudio.tags
+        
+        let audioCell = tableView.dequeueReusableCell(withIdentifier: "audio", for: indexPath) as! AudioRecordingCell
+        audioCell.fileNameLabel?.text = cellFileName
```

Comment:
You can have a method where you pass an AudioRecording and configure the cell there.